gitignore

```diff
@@ -0,0 +1 @@
+Bludidi/Hello_Microverse_Projects/test.md 
```

Comment:
Great job @Bludidi 🤝

- [ ] You may kindly remove the rest of the text before `test.md`. Only retain `test.md` because you want git to ignore the `test,md` file.
- [ ] Also kindly add the `node_modules` in this file as well. But put it on a different line.

Reply:
I have edited the file name I had put earlier to test.md  and also added node_modules in the gitignore.